services/src/main/java/org/fao/geonet/api/records/formatters/FormatterApi.java

```diff
@@ -302,7 +304,7 @@ public void getRecordFormattedBy(
                 context.getBean(DataManager.class).increasePopularity(context, String.valueOf(metadata.getId()));
             }
             writeOutResponse(context, metadataUuid,
-                LanguageUtils.locale2gnCode(locale.getISO3Language()),
+                isoLanguagesMapper.iso639_2B_to_iso639_2T(locale.getISO3Language()),
```

Comment:
To check with a previous similar code  that has been replaced by `isoLanguagesMapper.iso639_2T_to_iso639_2B` (also in the previous file `CatalogApi.java` and other files below)